clearly-defined/src/test/kotlin/ClearlyDefinedServiceTest.kt

```diff
@@ -49,6 +49,26 @@ import okio.Buffer
 private fun RequestBody.string() = Buffer().also { writeTo(it) }.readUtf8()
 
 class ClearlyDefinedServiceTest : WordSpec({
+    "Downloading a contribution patch" should {
+        "return curation data" {
+            val service = ClearlyDefinedService.create(Server.PRODUCTION)
```

Comment:
For the upload test non-production servers are used. Is it intended to use the production server? If so, how about adding a code comment providing a rationale?

Reply:
> Is it intended to use the production server?

Yes.

> If so, how about adding a code comment providing a rationale?

Done.

Reply:
Thanks!